docs/docs/commands/slashingImport.mdx

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 4
+id: slashing-import 
+---
+
+# slashing-import
+
+Running `sedge slashing-import` will import slashing protection data to the current validator client. The slashing protection data is a JSON file that meets with the [EIP-3076](https://eips.ethereum.org/EIPS/eip-3076) specification.
+
+## Help
+
+```shell
+$ sedge slashing-import --help
+
+Import Slashing Protection Interchange Format (EIP-3076) data. This command assumes
+that the validator client container exists, stopped or not and that its database
+is already initialized. The validator database is initialized if the validator is
+running or has already run but is stopped, and also after importing the validator keys.
+
+This command stops the validator client during the importing process due to the
+validator database being locked while it's running but leaves the validator client
+in the same state in which it was found. That means if the validator is running/stopped
+before the import, then the validator will be running/stopped after the command
+is executed, regardless of whether the export fails or not. To force a different
+behavior use --start-validator and --stop-validator flags.
+
+Usage:
+  sedge slashing-import [validator] [flags]
```

Comment:
Same as for Export doc

Reply:
Fixed here https://github.com/NethermindEth/sedge/pull/209/commits/3644f27e138267b8ca3c5b615210d9daa2098bf6